src/test/java/org/jenkinsci/plugins/saml/SamlSecurityRealmTest.java

```diff
@@ -168,13 +169,15 @@ public void testReadSimpleConfigurationAdvancedConfiguration() throws Exception
     }
 
     @LocalData("testHugeNumberOfUsers")
-    @Test(timeout = 15000)
+    @WithTimeout(240)
```

Comment:
Why reduced now?

Reply:
It's actually increased as @Test takes milliseconds and @WithTimeout takes seconds.  The timeout setting in @Test was being ignored anyway because of the way we run tests.

Reply:
Thanks for the clarification!

Reply:
`Test(timeout)` => milli-seconds
`WithTimeout` => seconds

So it's really an increase from 15sec to 4min ;)